Source: Imported from Gentoo
Upstream: no
Reason: Enable /etc/whois.conf support (Gentoo bug #89417)
--- config.h
+++ config.h
@@ -13,5 +13,3 @@
 /* Configuration file */
-/*
 #define CONFIG_FILE "/etc/whois.conf"
-*/
 
